feat: alert history dropdown functionality

ahmadshaheer opened the pull request:
### Summary
alert history dropdown functionality
<!-- ✍️ A clear and concise description...-->

#### Related Issues / PR's

<!-- ✍️ Add the issues being resolved here and related PR's where applicable  -->

#### Screenshots
![image](https://github.com/user-attachments/assets/48440786-38fc-4fe0-9a0f-e39729981353)

NA

<!-- ✍️ Add screenshots of before and after changes where applicable-->

#### Affected Areas and Manually Tested Areas

<!-- ✍️ Add details of blast radius and dev testing areas where applicable-->

srikanthccv commented:
@ahmadshaheer I couldn't find the main PR from the `feat/alert-history` page so I am adding the issues here

1. remove the severity label from the top level (this is redundant because you are already showing the severity with special colours)

![Screenshot 2024-09-02 at 10 17 44 PM](https://github.com/user-attachments/assets/4d609aaa-9619-4a41-b153-644c8fad84bb)


2. I find the alert number of redundant
![Screenshot 2024-09-02 at 10 18 57 PM](https://github.com/user-attachments/assets/1bff0dce-16bc-4495-a4a5-7af50d23962a)

3. add a description to disable because it's not clear what the toggle does. does it turn off history or does it turn off/disable alert
![Screenshot 2024-09-02 at 10 20 34 PM](https://github.com/user-attachments/assets/13b80ea4-1db0-4af0-8ae1-7d02f0871f02)

4. The alert turn-off doesn't reflect immediately; needs a refresh. please rename the "turn-off" to "disable" for consistency.

after disable

![Screenshot 2024-09-02 at 10 20 34 PM](https://github.com/user-attachments/assets/8d4e3e2e-7a93-4675-9b8f-5fca94c19244)

after refresh

![Screenshot 2024-09-02 at 10 24 44 PM](https://github.com/user-attachments/assets/3bfc37bc-deff-48d9-9aa5-e07a200231a5)

6. make the default period 6h in history

7. change the top contributors' description to
"Top contributors highlight the most frequently triggering group-by attributes in multi-dimensional alerts" because not every alert will need to have group-by.

8. Search doesn't work - we should either remove it or make it functional
9. The timeline doesn't show all the items (I am not sure what the logic but it should show all the items in /overall_status)
10. Let's not show the value column for now
11. on the next page offset should be the total consumed so far but `1` is sent. It should be like 20, 40, etc..
12. The logs link shows "View traces" and the traces link shows "View logs" link

ahmadshaheer commented:
> 7. change the top contributors' description to
"Top contributors highlight the most frequently triggering group-by attributes in multi-dimensional alerts" because not every alert will need to have group-by.

@srikanthccv, should we remove the **Configure Alert Rule** button as well?

CC: @sharpshooter90

ahmadshaheer commented:
>  1. remove the severity label from the top level (this is redundant because you are already showing the severity with special colours)

I removed the label containing the severity number
CC: @sharpshooter90 

> 2. I find the alert number of redundant 

I removed the the second id i.e. `id` infront of alert rule name
CC: @sharpshooter90 

>  3. add a description to disable because it's not clear what the toggle does. does it turn off history or does it turn off/disable alert

I added a tooltip saying Enable alert / Disable alert
CC: @sharpshooter90
